step.yml

```diff
@@ -100,19 +108,32 @@ inputs:
         - "no"
   - clone_depth:
     opts:
-      category: Clone Config
+      category: "Checkout options"
       title: "Limit fetching to the specified number of commits"
       description: |-
         Limit fetching to the specified number of commits.
         The value should be a decimal number, for example `10`.
   - limit_submodule_update_depth: "yes"
     opts:
-      category: Clone Config
+      category: "Checkout options"
       title: "Shallow update submodules"
       summary: "Should submodules be shallow cloned with a history truncated to the latest revisions."
       value_options:
         - "yes"
         - "no"
+  - merge_pr: "yes"
+    opts:
+      category: "Checkout options"
+      title: "Merge the Pull Request source branch into target"
+      summary: "Allows to validate a temporary merged state. Optional if head branch is required to be up-to-date before merging."
+      description: |-
+        Allows disabling merging the head and base branches, 
+        in case the git provider has an option to require the Pull Request branch to be up-to-date.
```

Comment:
Blocks merging the head and base branches, if the git provider has an option to require the pull request banch to be up-to-date.

I hope this is what you mean here.:)

Reply:
We do not block anything, just that this option allows to handle a PR as a push event (so no merge will occur as normally)

Reply:
Disables merging the head and base branches if the the git provider has an option to require the Pull Request branch to be up-to-date.